VisualStudio.gitignore

```diff
@@ -13,19 +13,22 @@
 # User-specific files (MonoDevelop/Xamarin Studio)
 *.userprefs
 
+# Mono auto generated files
+mono_crash.*
```

Comment:
@JFMG given the lack of clarity and documentation about where the other behaviour is coming from, are we able to focus on just this change? I'd be fine with future PRs if you uncover more context about the below rules.

Reply:
Yep, sure. I will update the PR

Reply:
@shiftkey Please review my latest PR